src/StarResults.js

```diff
@@ -90,6 +90,10 @@ function parseData(header, data) {
   const candidateOrder = sortCandidates(candidates, matrix);
   const singleResults = splitCandidates(candidateOrder, candidates, matrix);
   const multiResults = splitMulti(singleResults, candidates, matrix);
+  // The splitProportional method alters the candidate array it is passed
+  // so let's pass it a copy of the array
+  const propResults = splitProportional([...candidates], scores, 3);

```

Comment:
Rather than a hard-coded value of 3 winners, allow nWinners to be passed as an optional parameter to StarParse and StarResults.  If null, default to something like nWinners = ceiling(nCandidates/2)